drivers/hmis_external_apis/app/models/hmis_external_apis/ac_hmis/importers/loaders/referral_postings_loader.rb

```diff
@@ -294,6 +297,9 @@ def build_posting_records
         project_id = row_value(row, field: 'PROGRAM_ID')
         referral_id = row_value(row, field: 'REFERRAL_ID')
 
+        # There is one row per household member in the referral, but we only need to process one of them.
+        next if seen.include?(referral_id)
```

Comment:
iirc, I think I had this further down in the loop to log warning on the row. But that's probably just noise, moving it earlier makes sense